testing/bsg_fpu/div_srt_32/bsg_test_dpi.c

```diff
@@ -0,0 +1,21 @@
+#include<svdpi.h>
```

Comment:
It seems that you put the trace and result into a ROM file, and then using a C file to make comparison? If so, I think SystemVerilog DPI is very suitable for this situation.
I'm not sure whether DPI is banned in basejump team but it can really make fp test convenient.

Reply:
DPI is fine in general.  I think Tommy's concern is that the test shouldn't be tied to x86's implementation of fp.  But if they're both IEEE compliant, probably shouldn't cause an issue?

For what it's worth, http://www.jhauser.us/arithmetic/TestFloat.html is the gold standard for testing IEEE floating point

Reply:
@dpetrisko Thanks for your suggestion, I'll check it out and rewrite my test file!

Reply:
you are free to use whatever technology you prefer. my point is that all the corner cases and their exact expected outcome should be documented like in README (e.g. what happens when the input is denormal,  or what should be outputted when there is invalid operation, etc), so that when we run some serious randomized tests, we know exactly what to expect. Besides those corner cases, there should be no difference in computed result.

Reply:
@tommydcjung I get the point, thanks!